README.md

```diff
@@ -30,28 +30,18 @@ You will need the following things properly installed on your computer.
 
 * `git clone git@github.com/adfinis-sygroup/timed-frontend`
 * `cd timed-frontend`
-* `npm install`
-* `bower install`
-
-## Configurations
-
-If you wish to add custom exports to the analysis section, you can create an environment variable with the name `TIMED_REPORT_EXPORT` and add a JSON Array as value.
-```
-TIMED_REPORT_EXPORT=[{"label":"Special Export","url":"/api/v1/special/export"}]
-```
-In development you can create a `.env` file and add the environment variable.
-```
-echo 'TIMED_REPORT_EXPORT=[{"label":"Special Export","url":"/api/v1/special/export"}]' > .env
-```
+* `yarn`
 
 ## Running / Development
 
-* `ember serve`
+* `ember server`
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 
 If you have a running [backend](https://github.com/adfinis-sygroup/timed-backend) you need to run
 
 * `ember serve --proxy=http://localhost:8000`
```

Comment:
Also change to `ember server` for consistency

Reply:
Done